### ISSUE TITLE
A11y_SQL Azure Data Studio_Add an Azure account_Add an Azure account_Screenreader:Screenreader does not announce the heading level and name for Linked account heading

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/azuredatastudio#15836]
"[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier." 

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.6;#NVDA;#JAWS;

### Environment Details:
Application Name: SQL Azure Data Studio 
Version: 1.30.0-insider (user setup)
Windows Version: Win10
Screen reader : JAWS, NVDA2020.2

### Repro Steps:

1. Launch the application-Azure Data Studio.
2.  Select the Connections icon (if not already selected) from left bar.
3. Hover over to the 'Azure' header and select the '+' icon.
4.  Hover over to the Azure header in the blade that opens up and click on the "+" icon.
5.  Open a browser and enter the url as shown in the Website field and then the user code provided in the User Code field. You can also click on the Copy & Open button and select "Open" on the dialog when promped for opening the microsoft.com/devicelogin url, and paste the code (already in the clipboard).
6. Start screenreader and tab navigate to Linked account heading element and observe its announce the heading name and level.

### Actual:
Screenreader does not announce the heading level and name for Linked account heading.

**This issue is observed with both NVDA and JAWS screen readers.**

### Expected:
Screenreader should announce the heading level and name for Linked account heading.

### User Impact:
Screenreader user will miss some information regarding heading name and level.

**Comment by alanrenmsft** [comment]
@SwatiTiwari-TCS could you please attach a video for the issue?

**Comment by Harika1702** [comment]
![15836_A11y_Issue fixed](https://user-images.githubusercontent.com/100270074/217773327-d59c90db-42b5-48e9-8c2c-e4d7d170bd47.jpg)
Verified this issue in the below environment.
App name: Azure Data studio.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623.
GitHub Tags: #Closed;
